src/builtins/Transformers.jl

```diff
@@ -847,9 +847,9 @@ function MMI.fit(transformer::OneHotEncoder, verbosity::Int, X)
     end
 
     ref_name_pairs_given_feature =
-        Dict{Symbol,Vector{Pair{<:Unsigned,Symbol}}}()
+        Dict{Symbol,Vector{Union{Pair{<:Unsigned,Symbol}, Pair{Missing, Symbol}}}}()
```

Comment:
I'm a little confused by this expansion of the type. In `fit` it appears that you will never actually encounter a pair of the type `Pair{Missing, Symbol}`. See line 861. In `fit` you only really care about `levels(col)` not actual elements of `col`. Rather, it is only in `transform` that elements matter. 

Am I missing something here?

Reply:
Oh yes. I almost forgot to remove this line. Turns out that I wrote this for doing [this](https://github.com/JuliaAI/MLJModels.jl/issues/450#issuecomment-1100424642).